ansible/roles/products/common/docker-container/tasks/reinstall.yml

```diff
@@ -175,13 +175,23 @@
     - "'arm_' in product"
     - skip_molecule_task is not defined
 
+- name: Set documentation variables for arm and glove
+  set_fact:
+    documentation_repo: sr_dexterous_hand_glove_documentation
+    readthedocs_token: "{{ glove_hand_readthedocs_token }}"
+    documentation_product: dexterous-hand-and-glove
+  when:
+    - "'arm_' in product"
+    - skip_molecule_task is not defined
+
 - name: Update documentation inside the docker container for teleop or arm+hand
   import_tasks: modify-container.yml
   vars:
     modify_script_name: "update_documentation.sh"
     modify_script_commands: "roscd {{ documentation_repo }};curl -H 'Authorization: Token {{ readthedocs_token }}' https://shadow-robot-company-{{ documentation_product }}.readthedocs-hosted.com/_/downloads/en/latest/htmlzip/ -o temp.zip;unzip temp.zip;mv shadow-robot-company-{{ documentation_product }}-latest/ html/"
   when:
-    - "glove == 'haptx' or glove == 'shadow_glove' or 'arm_' in product"
+    - "glove == 'haptx' or glove == 'shadow_glove' or 'arm_' in product or 'glove_' in product"
```

Comment:
aren't the first 2 condition redundant then glove_ in product?

Reply:
Not for teleop :)